scripts/ci/install_docker.sh

```diff
@@ -5,11 +5,7 @@ if [[ $(dpkg --print-architecture) == "amd64" ]]; then
     echo "Docker is already installed on AMD64"
     exit 0
 fi
-# https://docs.docker.com/engine/security/rootless/
```

Comment:
Not able to install rootless docker in 22.04 and 24.04.

Ubuntu use cgroup v2 after 22.04. Docker keeps raising `docker: Error response from daemon: open /sys/fs/cgroup/user.slice/user-1000.slice/cgroup.controllers: no such file or directory` when run docker, but `/sys/fs/cgroup/user.slice/` exists. I guess `user-1000` is logged in and this folder does not exist.

Reply:
The document https://docs.docker.com/engine/security/rootless/ still exists and is valid. Is `user-1000.slice`-not-existing an ADO-specific issue?

Reply:
Yes. Some say `user-1000.slice` is created when a user logs in or has a running process, but I can't find any reference.